lesson_05/kaliuzhna_05.py

```diff
@@ -0,0 +1,80 @@
+# 1. Inform user that such currencies pair is not supported yet.
+# 2. Be sure that `currency_from_val` positive integer or float
+#    2.1 Inform user that such value is incorrect
+# 3. New function which adds new pairs with ratio to `currencies_map`:
+#    function name: update_currencies_map
+#    with params: currency_from, currency_to, currency_from_val
+#    Inform user about results
+
+
+
+currencies_map = {
+    "USD_UAH": 40.7,
+    "UAH_USD": 0.027,
+}
+
+
+def currency_converter(currency_from, currency_to, currency_from_val):
+    counter = 0
+    if type(currency_from) == str:
+        if currency_from.isalpha():
+            if len(currency_from) == 3:
+                if currency_from.upper() == currency_from:
+                    counter += 1
+                else:
+                    print(f"Currency_from: {currency_from} is not upper")
+            else:
+                print(f"Length of currency_from: {currency_from} is not equal to 3 characters")
+        else:
+            print(f"Currency_from: {currency_from} is not alpha")
+    else:
+        print(f"Currency_from: {currency_from} isn`t string")
+    if type(currency_to) == str:
```

Comment:
you do the same validation steps for both `currency_from` and `currency_to` - it breaks the DRY principle
in this case we can use iteration:
```python
for currency in [currency_from, currency_to]:
    # some validation steps
```